network/gossip/libp2p/test/testUtil.go

```diff
@@ -24,16 +25,53 @@ import (
 	"github.com/onflow/flow-go/utils/unittest"
 )
 
-var rootBlockID = unittest.IdentifierFixture().String()
+var rootBlockID string
+
+// allocatedPorts keeps track of ports allocated to different tests
+var allocatedPorts map[int]struct{}
+
+// mu makes allocatedPorts access concurrency safe
+var mu sync.Mutex
```

Comment:
I would encapsulate this with the ports mapping into a structure to make it more clear they are coupled. It looks like `test` is a fairly large package, so it's best to minimize and consolidate package-global variables.

Reply:
fixed in https://github.com/onflow/flow-go/pull/99/commits/9e2fc91ca8506022a22c721bec7b415240f9c153.